lib/rawscsi/request_signature.rb

```diff
@@ -1,9 +1,8 @@
 module Rawscsi
   class RequestSignature
-    def initialize(*args)
-      options = args.extract_options!
+    def initialize(options)
       required_attributes_missing = []
-      require_attribute = -> (name) do
+      require_attribute = lambda do |name|
```

Comment:
@ivan-kolmychek  I changed the syntax here since `->` isn't supported in Ruby 1.9.3